com.github.quaternion.json

```diff
@@ -14,15 +14,25 @@
         "--device=dri"
     ],
     "modules": [
+        {
+            "name": "libqmatrixclient",
```

Comment:
This will install headers and such that need to be added to `cleanup`

Reply:
Done in 669067c